src/themes/default/elements/icon.overrides

```diff
@@ -82,8 +84,24 @@ i.icon.at:before { content: "\f1fa"; }
 i.icon.eyedropper:before { content: "\f1fb"; }
 i.icon.paint.brush:before { content: "\f1fc"; }
 i.icon.heartbeat:before { content: "\f21e"; }
+i.icon.mouse.pointer:before { content: "\f245"; }
+i.icon.hand.pointer:before { content: "\f25a"; }
+i.icon.hand.peace:before { content: "\f25b"; }
+i.icon.trademark:before { content: "\f25c"; }
+i.icon.registered:before { content: "\f25d"; }
+i.icon.creative.commons:before { content: "\f25e"; }
+i.icon.add.to.calendar:before { content: "\f271"; }
+i.icon.remove.from.calendar:before { content: "\f272"; }
+i.icon.delete.calendar:before { content: "\f273"; }
+i.icon.checked.calendar:before { content: "\f274"; }
+i.icon.industry:before { content: "\f275"; }
+i.icon.shopping.bag:before { content: "\f290"; }
+i.icon.shopping.basket:before { content: "\f291"; }
+i.icon.hashtag:before { content: "\f292"; }
+i.icon.percent:before { content: "\f295"; }
```

Comment:
New icons for the Web Content category: `shopping bag`, `shopping basket`, `hashtag` (the # char) and `percent` (yes, the % char)